### PR TITLE
Fix path of "enable version 1.7 on RTD:"

### PR DESCRIPTION
**Pull request by tkoyama010**
Hi. I fixed path of "enable version 1.7 on RTD:". It WFM. Thank you for reading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,5 +90,5 @@ After that, you should commit updated po files.
 
 4. enable version 1.7 on RTD:
 
-   https://readthedocs.org/dashboard/sphinx-ja/versions/
+   https://readthedocs.org/projects/sphinx/versions/
```
Comment (tk0miya) [inline README.md]: @shimizukawa Could you check this please? Surely `/dashboard/sphinx-ja/versions` is NOT FOUND. But I'm not sure `/projects/sphinx/versions` is correct for this case...

Comment (shimizukawa) [reply README.md]: Recent updates, ReadTheDocs changes URL structure, so `dashboard` is deprecated. 

`/projects/sphinx/versions/` and
`/projects/sphinx-<lang>/versions` are correct URLs.

In this section, as a `sphinx-doc-translations` repository, `-<lang>` is important I think.
So I recommend:
```suggestion
   https://readthedocs.org/projects/sphinx-ja/versions/
```

**Comment by tkoyama010** [comment]
Oops! I forgot to add <language>. Thank you for your review.

**Comment by tkoyama010** [comment]
thx

**Pull request merged**